### PR TITLE
docs: Improve "updating angular cli" steps

### PR DESCRIPTION
**Pull request by dmytroyarmak**
Add step that removes "angular-cli" from "package.json" to make sure that after
upgrading from "angular-cli" to "@angular/cli" only "@angular/cli" is installed
into "node_modules".

It's needed to avoid problems with npm run scripts that uses "ng" command.

Otherwise, there will be an error ("You have to be inside an angular-cli project in
order to use the test command.") when running "npm start", "npm test" or
"npm run ng -- ..." commands.

Here are related issues: #4463, #4439

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -404,6 +404,7 @@ npm install -g @angular/cli@latest
 Local project package:
 ```bash
 rm -rf node_modules dist # use rmdir on Windows
+npm uninstall --save-dev angular-cli
```
Comment (dmytroyarmak) [inline README.md]: But lines 399-401 are about cleaning and installing "global package" and this one is about the "local project package" .

**Comment by hansl** [comment]
This is a temporary pain, but should not be part of the update steps. Could you rephrase it to say something like "if you're using the old `angular-cli`, please do `npm uninstall -g angular-cli`" ?

**Comment by dmytroyarmak** [comment]
Thank you @hansl 
I've separated removing of `angular-cli` to separate steps. Does it sound better now?
